### ISSUE TITLE
cosine logits computation

### ISSUE THREAD

**Issue by hbw2355265** [JungHunOh/CLOSER_ECCV2024#4]
I have some questions about lines 36-39 in help.py. Do you use model.encoder instead of encode, so that the feats dimension is (B,C,H,W) How to calculate with classifier weights

**Comment by JungHunOh** [comment]
Hi.
model.encoder produces an output with dimension of (B,d) where d is the feature dimension.
Could you please check it again?
